pkgs/development/tools/gomplate/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
You should be able to switch to `buildGoModule` as that's the recommended approach for projects that support it.